htlcswitch/hop/error_encryptor.go

```diff
@@ -88,9 +88,9 @@ type SphinxErrorEncrypter struct {
 // should be used to deserialize an encoded SphinxErrorEncrypter. Since the
 // actual encrypter is not stored in plaintext while at rest, reconstructing the
 // error encrypter requires:
-//   1) Decode: to deserialize the ephemeral public key.
-//   2) Reextract: to "unlock" the actual error encrypter using an active
-//        OnionProcessor.
+//  1. Decode: to deserialize the ephemeral public key.
```

Comment:
It's pretty nice that we now get more formatting support in comments with 1.18. Though it seems the auto-formatter for the comments is pretty opinionated...

Reply:
haha yeah, I'm a bit surprised to see how strict the formatter is, apparently it has its own taste...

Reply:
Yeh I've tried to fight it a bit after I upgraded to 1.19 locally, unsuccessfully. The main thing that is weird at times is the forced spacing on nested lists.